### ISSUE TITLE
problem in  evaluate.py 

### ISSUE THREAD

**Issue by aijianiula0601** [mozilla/DeepSpeech#1985]
there's a place confuses me when i read there evaluate.py's codes. 

The code in evaluate.py  below:

----------------------------------------------------------------------------------------------------
    for logits, batch in bar(zip(logitses, split_data(test_data, FLAGS.test_batch_size))):
        seq_lengths = batch['features_len'].values.astype(np.int32)
        decoded = ctc_beam_search_decoder_batch(logits, seq_lengths, Config.alphabet, FLAGS.beam_width, num_processes=num_processes, scorer=scorer)
        ground_truths.extend(Config.alphabet.decode(l) for l in batch['transcript'])
        predictions.extend(d[0][1] for d in decoded)

-----------------------------------------------------------------------------------------------------

why  the code  "predictions.extend(d[0][1] for d in decoded)"  that the predictitions always only extend d[0][1]?  the default value of  beam_width is 1024, but predititons extend d[0][1] which is mean beam_width=1 not 1024,is it right?

**Comment by kdavis-mozilla** [comment]
We try to reserve issues for bugs or feature enhancements, and as this does not appear to be a bug or feature enhancement (more a discussion of the code), we ask that you move this discussion to [Discourse](https://discourse.mozilla.org/c/deep-speech) please. Thank you.
